### PR TITLE
Balancer console

### PR DESCRIPTION
**Pull request by agneum**
- init console interface for Balancer App
- support commands:
   - show key ranges
   - split key range
- use spqrlog package to log messages

Examples:
### Connection
```
$ psql "host=::1 port=8433 dbname=db1 sslmode=disable user=user1"
Welcome
psql (14.2 (Ubuntu 14.2-1.pgdg20.04+1), server console)
Type "help" for help.
```
### Show key ranges
```
db1=?> show key_ranges;
 Key range ID | Shard ID | Lower bound | Upper bound 
--------------+----------+-------------+-------------
 krid1        | 1        | 1           | 1000
 krid2        | 2        | 1000        | 2000
(2 rows)
```
### Split key ranges
```
db1=?> split key range krid3 from krid2 by 1500;
        split key range        
-------------------------------
 split key range krid2 by 1500
(1 row)

db1=?> show key_ranges;
 Key range ID | Shard ID | Lower bound | Upper bound 
--------------+----------+-------------+-------------
 krid1        | 1        | 1           | 1000
 krid2        | 2        | 1000        | 1500
 krid3        | 2        | 1500        | 2000
(3 rows)
```

### CODE REVIEW

File: coordinator/provider/keyranges.go [hunk 1]
```diff
@@ -62,13 +63,36 @@ func (c CoordinatorService) KeyRangeIDByBounds(ctx context.Context, keyRange *pr
...
 	if err != nil {
 		return nil, err
 	}
 
+	splitKR := &kr.SplitKeyRange{
+		Bound: request.Bound,
+		Krid:  request.KeyRangeInfo.Krid,
+	}
+
+	// TODO: choose a key range without hacks.
```
Comment (reshke) [inline coordinator/provider/keyranges.go:76]: lets just explisitly pass source key range here (and regenerate protos accordingly)

File: pkg/clientinteractor/interactor.go [hunk 2]
```diff
@@ -3,13 +3,17 @@ package clientinteractor
 import (
 	"context"
 	"fmt"
+	"net"
+
+	"github.com/wal-g/tracelog"
```
Comment (reshke) [inline pkg/clientinteractor/interactor.go:8]: pls use spqrlog

File: pkg/clientinteractor/interactor.go [hunk 3]
```diff
@@ -119,17 +123,75 @@ func (pi *PSQLInteractor) AddShard(cl client.Client, shard *datashards.DataShard
 }
 
 func (pi *PSQLInteractor) KeyRanges(krs []*kr.KeyRange, cl client.Client) error {
-	spqrlog.Logger.Printf(spqrlog.DEBUG1, "listing key ranges")
-	if err := pi.WriteHeader("listing key ranges", cl); err != nil {
-		spqrlog.Logger.PrintError(err)
-		return err
+	tracelog.InfoLogger.Printf("listing key ranges")
```
Comment (reshke) [inline pkg/clientinteractor/interactor.go:126]: spqrlog.Logger.Printf(spqrlog.DEBUG3, ...)

or debug2 or smth

File: coordinator/provider/keyranges.go (hunk 1 above)
Comment (agneum) [reply coordinator/provider/keyranges.go]: Fixed

File: pkg/clientinteractor/interactor.go (hunk 2 above)
Comment (agneum) [reply pkg/clientinteractor/interactor.go]: Done

File: pkg/clientinteractor/interactor.go (hunk 3 above)
Comment (agneum) [reply pkg/clientinteractor/interactor.go]: Done

**Review by reshke — APPROVED** [review]

**Pull request merged**